gulpfile.js

```diff
@@ -653,6 +653,26 @@ const copyMultiplayerHtml = () => rimraf("webapp/public/multiplayer.html")
 
 const multiplayer = gulp.series(cleanMultiplayer, buildMultiplayer, gulp.series(copyMultiplayerCss, copyMultiplayerJs, copyMultiplayerHtml));
 
+/********************************************************
+                 Webapp build wrappers
+*********************************************************/
+
+const shouldBuildWebapps = () => (process.argv.indexOf("--no-webapps") === -1);
```

Comment:
Nit; making this an async function would remove the need for an explicit returned noop / inverted logic, but fine either way

Reply:
Something we could improve later for sure. Gulp's expectations here are very specific and I wrestled with it more than I'd expected to.